.github/workflows/test.yml

```diff
@@ -0,0 +1,57 @@
+name: Validate 'setup-go'
+on: 
+  push:
+  pull_request:
+  schedule:
+    - cron: 0 0 * * *
+
+jobs:
+  default-version:
+    name: Setup default version
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-16.04, ubuntu-18.04]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: setup default go 
+      uses: ./
+
+    - name: Validate version
+      run: go version
+
+    - name: Run simple go code
+      run: go run __tests__/simple/simple.go
+
+  setup-versions-from-manifest:
+    name: Setup ${{ matrix.go }} ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-16.04, ubuntu-18.04]
+        go: [1.11.12, 1.11.13, 1.12.17, 1.13.12, 1.14.4]
```

Comment:
As far as I understand in scope of this test we need to validate versions downloading from the manifest. `1.11.13, 1.12.17, 1.13.12,  1.14.4` are available in the local cache. I suggest to specify the following versions:  `1.12.16, 1.13.11,  1.14.3`

Reply:
Done.